content/guides/clj_datatype_constructs.adoc

```diff
@@ -0,0 +1,153 @@
+= Understanding Clojure's Polymorphism
```

Comment:
Updated title.

I do mention deftype, but not in depth believing that by default, one should reach for records.
Feedback would be appreciated though.